src/mcp_atlassian/jira/sprints.py

```diff
@@ -68,3 +68,44 @@ def get_all_sprints_from_board_model(
             limit=limit,
         )
         return [JiraSprint.from_api_response(sprint) for sprint in sprints]
+
+    def create_sprint(
+        self,
+        board_id: str,
+        sprint_name: str,
+        start_date: str,
+        end_date: str,
+        goal: str = None,
+    ) -> JiraSprint:
+        """
+        Create a new sprint.
+
+        Args:
+            sprint_name: Sprint name
+            board_id: Board ID
+            start_date: Start date in ISO format
+            end_date: End date in ISO format
+            goal: Sprint goal
+
+        Returns:
+            Created sprint details
+        """
```

Comment:
The parameter order in the docstring (`sprint_name`, `board_id`) does not match the actual function signature (`board_id`, `sprint_name`). Please reorder the docstring parameters to match the function definition for clarity and consistency.